NServiceBus.Transport.AzureServiceBus 3.1.0 Updates (#5923)
Particular/docs.particular.net: transports/azure-service-bus/operational-scripting.md

                              [--size]
                              [--partitioned]
                              [--topic]
                              [--topic-to-publish-to] [--topic-to-subscribe-on]
                              [--subscription]
```

#### options

`-c` | `--connection-string` : Overrides the environment variable 'AzureServiceBus_ConnectionString'

> `-n` | `--namespace` : Sets the fully qualified namespace for connecting with cached credentials, such as those from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.

Reviewer: ```suggestion
`-n` | `--namespace` : Sets the fully qualified namespace to connect with cached credentials, e.g., credentials from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.
```

`-c` | `--connection-string` : Overrides the environment variable 'AzureServiceBus_ConnectionString'

> `-n` | `--namespace` : Sets the fully qualified namespace for connecting with cached credentials, such as those from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.

Reviewer: ```suggestion
`-n` | `--namespace` : Sets the fully qualified namespace to connect with cached credentials, e.g., credentials from Azure PowerShell or CLI. This setting cannot be used in conjunction with the connection string setting.
```
